ingestion/tests/integration/data_quality/test_data_diff.py

```diff
@@ -52,6 +54,9 @@
 from metadata.workflow.data_quality import TestSuiteWorkflow
 from metadata.workflow.metadata import MetadataWorkflow
 
+if not sys.version_info >= (3, 9):
+    pytest.skip("requires python 3.9+", allow_module_level=True)
+
```

Comment:
is that because of something in the test itself or because of something we do in the test case validation?

Reply:
related to incompatibility with testcontainers. add this information in the message.